### ISSUE TITLE
Add SAREF alignment in Rec doc

### ISSUE THREAD

**Issue by maximelefrancois86** [w3c/sdw-sosa-ssn#125]
I volunteer to work on an alignment of SOSA/SSN with ETSI SAREF (Smart Applications REFerence ontology) https://saref.etsi.org/core/

**Comment by dr-shorthair** [comment]
That would be appreciated.

**Comment by rob-metalinkage** [comment]
Hi Maxime

please note also an activity in the OGC space based on profiling SOSA and other ontologies w.r.t. SAREF 

https://github.com/opengeospatial/Agriculture-DWG/tree/main/aim_swg_charter

This will provide opportunities to test and promote such integrations, and will extend beyond the current joint WG charter.

**Comment by dr-shorthair** [comment]
This should be a priority now, particularly to also support the Usage Report for Actuation use-cases as well as the SOSA updates triggered by SAREF.

**Comment by dr-shorthair** [comment]
Any progress here @maximelefrancois86 ?

**Comment by maximelefrancois86** [comment]
Please review #206

**Comment by dr-shorthair** [comment]
Closed by #206

**Comment by dr-shorthair** [comment]
Issues that have [arisen as a result of work on PropertyOfInterest](https://github.com/w3c/sdw-sosa-ssn/pull/274#issuecomment-2529968944)

> Also https://raw.githack.com/w3c/sdw-sosa-ssn/252-remove-propertyofinterest/ssn/index.html#SAREF-alignment - I'm not sure that we have the `hasInput`, `hasOutput`, `hasInputValue` alignments correct yet, also `isPropertyOf` and `isPropertyOfInterestOf`. In particular, do we need the intermediate terms in the http://www.w3.org/ns/sosa/saref# namespace?

**Comment by dr-shorthair** [comment]
Current state of the SAREF alignment: https://w3c.github.io/sdw-sosa-ssn/ssn/#SAREF-alignment

**Comment by dr-shorthair** [comment]
The artefacts that need to be updated are 
- https://github.com/w3c/sdw-sosa-ssn/blob/gh-pages/ssn/chapters/SAREF-alignment.html 
- https://github.com/w3c/sdw-sosa-ssn/blob/gh-pages/ssn/rdf/ontology/alignments/sosa-saref.ttl

**Comment by dr-shorthair** [comment]
@maximelefrancois86 could you take a look at this please?

**Comment by maximelefrancois86** [comment]
Yes at the very end of Jan / beginning of Feb, we shall have SAREF v4 ready and I'll propagate things to here.

**Comment by dr-shorthair** [comment]
Hey @maximelefrancois86 did SAREF v4 get issued? 
Could you take a look at the alignment?

**Comment by maximelefrancois86** [comment]
Yes I should be able to finalize this this week

**Comment by dr-shorthair** [comment]
Should SAREF be cited in the Introduction to SSN as one of the key influences over SSN/SOSA ? - https://w3c.github.io/sdw-sosa-ssn/ssn/#Introduction

**Comment by maximelefrancois86** [comment]
I updated the alignment turtle file in the related branch, I'm waiting for feedback.
Next step is to update the html.
I'll try to edit some part of the intro as per your suggestion

**Comment by dr-shorthair** [comment]
https://github.com/w3c/sdw-sosa-ssn/blob/e4f57c494f2981c646780695fc877b03379cf8a2/ssn/rdf/ontology/alignments/sosa-saref.ttl#L106
```
sosa:hasFeatureOfInterest
  rdfs:subPropertyOf saref:actsUpon . # this also covers sosa:isFeatureOfInterestOf

sosa:hasUltimateFeatureOfInterest
  rdfs:subPropertyOf saref:actsUpon . # this also covers sosa:isUltimateFeatureOfInterestOf
```
hasFoI etc applies to all executions, including actuations, observations, samplings, - i.e. it is the more general case. 
So should the sub-property axiom be reversed? 

https://github.com/w3c/sdw-sosa-ssn/blob/e4f57c494f2981c646780695fc877b03379cf8a2/ssn/rdf/ontology/alignments/sosa-saref.ttl#L112
```
sosa:forProperty
  rdfs:subPropertyOf saref:actsUpon .
```
Shouldn't the alignment involve `sosa:actsOn` ? 

https://github.com/w3c/sdw-sosa-ssn/blob/e4f57c494f2981c646780695fc877b03379cf8a2/ssn/rdf/ontology/alignments/sosa-saref.ttl#L118
```
sosa:hasProcedure 
  rdfs:subPropertyOf saref:isActedUponBy .
```
`sosa:hasProcedure` covers actuatable and observable properties. So maybe the sub-property axiom should go the other way round?

**Comment by maximelefrancois86** [comment]
Answers to @dr-shorthair 

> hasFoI etc applies to all executions, including actuations, observations, samplings, - i.e. it is the more general case.
> So should the sub-property axiom be reversed?

`saref:actsUpon` is even more general than that, as it not only link all executions to FoI

> Shouldn't the alignment involve sosa:actsOn ?

no error here. In SAREF, actsUpon generalizes observes/controls

> `sosa:hasProcedure` covers actuatable and observable properties. So maybe the sub-property axiom should go the other way round?

actsUpon generalizes observes/controls

**Comment by dr-shorthair** [comment]
The tabulation in the HTML looks good. 
I suggest a sentence or two at the end of paragraph 3 of the Introduction acknowledging the contribution of SAREF to SSN/SOSA and the new edition would suit. 
- https://w3c.github.io/sdw-sosa-ssn/ssn/#Introduction
